From 6fe89706a22d9f614c9d0017d03cc8da6e5b1630 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Tue, 20 Jun 2023 05:27:51 +0800
Subject: [PATCH 30/30] Revert "drm/fb-helper: set x/yres_virtual in
 drm_fb_helper_check_var"

This reverts commit dc110b20f4ce752c232ad710a428dd97cf138338.
---
 drivers/gpu/drm/drm_fb_helper.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/gpu/drm/drm_fb_helper.c b/drivers/gpu/drm/drm_fb_helper.c
index 04f2ec2254e9f..ac5d61e65124e 100644
--- a/drivers/gpu/drm/drm_fb_helper.c
+++ b/drivers/gpu/drm/drm_fb_helper.c
@@ -1299,9 +1299,6 @@ int drm_fb_helper_check_var(struct fb_var_screeninfo *var,
 		return -EINVAL;
 	}
 
-	var->xres_virtual = fb->width;
-	var->yres_virtual = fb->height;
-
 	/*
 	 * Workaround for SDL 1.2, which is known to be setting all pixel format
 	 * fields values to zero in some cases. We treat this situation as a
-- 
2.25.1

